packages/rum/wdio.conf.js

```diff
@@ -62,16 +56,16 @@ const browserList = [
 const sauceConnectOpts = getSauceConnectOptions()
 
 exports.config = {
-  specs: [path.join(__dirname, '/test/e2e/**/*.e2e-spec.js')],
+  specs: [join(__dirname, '/test/e2e/standalone-html/*.e2e-spec.js')],
```

Comment:
Please ignore this for now. I will remove this once the code review is done.

Reply:
Do you want to revert this change?

Reply:
Sure, on it.